pypeit/spectrographs/keck_kcwi.py

```diff
@@ -130,6 +130,10 @@ def config_specific_par(self, scifile, inp_par=None):
             par['calibrations']['wavelengths']['method'] = 'full_template'
             par['calibrations']['wavelengths']['reid_arxiv'] = 'keck_kcwi_BM.fits'
             par['calibrations']['wavelengths']['lamps'] = ['FeI', 'ArI', 'ArII']
+        elif self.get_meta_value(headarr, 'dispname') == 'BL':
```

Comment:
looks like there might be some code replication in this
`if/else` block.  Reduce if possible.

Reply:
Done!